.github/workflows/release.yaml

```
@@ -0,0 +1,71 @@
+        id: tag_publish
+        run: |
+          pnpx lerna version ${{ github.event.inputs.release_type }} --conventional-commits --yes --no-private --force-publish --ignore-scripts
+          pnpx lerna publish from-git --yes --dist-tag ${{ github.event.inputs.release_type == 'preminor' && 'next' || 'latest' }}
+
+      - name: Get Version Tag
+        id: get_tag
+        run: echo "TAG=$(git describe --tags --abbrev=0)" >> $GITHUB_OUTPUT
+
+      - name: Generate Release Body
+        id: release_body
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:38: Double quote to prevent globbing and word splitting

(shellcheck)